lib/l10n/intl_en.arb

```diff
@@ -55,6 +59,8 @@
   "sectionEventsComingUp": "Events coming up",
   "sectionFAQ": "FAQ",
   "sectionGrading": "Grading",
+  "sectionFeedbackTODO": "Feedback To-Do",
```

Comment:
Maybe "Feedback to complete"? And I'd change the string key as well, it sounds kinda bad.